packages/mgt/src/components/mgt-get/mgt-get.ts

```diff
@@ -219,40 +254,57 @@ export class MgtGet extends MgtTemplatedComponent {
           request = request.middlewareOptions(prepScopes(...this.scopes));
         }
 
-        let response = await request.get();
+        let response = null;
+        if (this.type == ResponseType.json) {
```

Comment:
Use `===` instead of `==` for type safety :)

```suggestion
        if (this.type === ResponseType.json) {
```